index.html

```diff
@@ -1,22 +1,59 @@
 <!DOCTYPE html>
-<html>
+<ht

```

Comment:
can we please have the html tag in one line only

Reply:
Implemented in eb31b00fe595c41cf526af9afff9453fb6aa9840